docs/content/ingestion/native_tasks.md

```diff
@@ -0,0 +1,523 @@
+---
+layout: doc_page
+---
+# Native Index Tasks
+
+Druid currently has two types of native index tasks, i.e., `local index task` and `single phase parallel index task`.
```

Comment:
nit: can you change the order to match the order of the following descriptions and maybe make a bit more descriptive? Maybe something like this?

>Druid currently has two types of native batch indexing tasks, `index_single_phase_parallel ` which runs tasks remotely and in parallel on middle manager nodes, and `index` which will run a single indexing task locally on the overlord.

Reply:
Good point. But, the `index` task also runs on a middleManager. I changed like below.

```
Druid currently has two types of native batch indexing tasks, `index_single_phase_parallel` which runs tasks
in parallel on multiple middle manager nodes, and `index` which will run a single indexing task locally on a single
middle manager.
```